LeaderboardBackend/Controllers/UsersController.cs

```diff
@@ -36,11 +36,22 @@ public async Task<ActionResult<User>> GetUser(Guid id)
 	[HttpPost("register")]
 	public async Task<ActionResult<User>> Register([FromBody] RegisterRequest body)
 	{
+		// This shouldn't hit normally, since we have the CompareAttribute in Register.cs
 		if (body.Password != body.PasswordConfirm)
 		{
 			return BadRequest();
 		}
 
+		if (await _userService.GetUserByEmail(body.Email) != null)
+		{
+			return Conflict("A user already exists with this email.");
```

Comment:
So let's leave this here, but I think the plan overall is not to use `409 Conflict` here worth keeping in mind

Reply:
Tru. I'll add a comment. In fact; the `MiddlewareResultHandler` I've written in [the authZ PR](https://github.com/leaderboardsgg/leaderboard-backend-poc/pull/60/files#diff-a1f7e3047926dcd210c9097f55ef957ce5234adf163f98e2d53cf4d9570ccea8R30-R32) should be able to override this, if I'm not mistaken.